pynicotine/pluginsystem.py

```diff
@@ -506,37 +507,33 @@ def enable_plugin(self, plugin_name):
                 if not data:
                     continue
 
-                command = "/" + command
                 disabled_interfaces = data.get("disable", [])
 
                 if "group" not in data:
                     # Group commands under human-friendly plugin name by default
                     data["group"] = plugin.human_name
 
-                for command_interface, command_list in (
-                    ("chatroom", self.chatroom_commands),
-                    ("private_chat", self.private_chat_commands),
-                    ("cli", self.cli_commands)
-                ):
+                for command_interface, command_list in self.commands.items():
                     if command_interface in disabled_interfaces:
                         continue
 
                     if command in command_list:
                         log.add(_("Conflicting %(interface)s command in plugin %(name)s: %(command)s"),
-                                {"interface": command_interface, "name": plugin.human_name, "command": command})
+                                {"interface": command_interface, "name": plugin.human_name, "command": f"/{command}"})
                         continue
 
                     command_list[command] = data
 
-            for command, _func in plugin.__publiccommands__:
-                command = "/" + command
-                if command not in self.chatroom_commands:
-                    self.chatroom_commands[command] = None
+            # Legacy commands
+            for command_interface, plugin_commands in (
+                ("chatroom", plugin.__publiccommands__),
+                ("private_chat", plugin.__privatecommands__)
+            ):
+                interface_commands = self.commands.get(command_interface)
```

Comment:
Set to an empty list `[]` by default instead of `None`, to avoid possible NoneType crash on line 535.